src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/SpecificPriceController.php

```diff
@@ -182,8 +182,15 @@ private function formatSpecificPricesList(SpecificPriceList $specificPriceListFo
                 'country' => $specificPrice->getCountryName() ?? $this->trans('All countries', 'Admin.Global'),
                 'group' => $specificPrice->getGroupName() ?? $this->trans('All groups', 'Admin.Global'),
                 'customer' => $specificPrice->getCustomerName() ?? $this->trans('All customers', 'Admin.Global'),
-                'price' => $this->formatPrice($specificPrice->getFixedPrice()),
-                'impact' => $this->formatImpact($specificPrice->getReductionType(), $specificPrice->getReductionValue()),
+                'price' => $this->formatPrice(
+                    $specificPrice->getFixedPrice(),
+                    $specificPrice->getCurrencyISOCode() ?: $this->getContextCurrencyIso()
```

Comment:
Oh tricky as well! This originally "simple" PR keeps getting harder and harder 😅

Reply:
I confirm. But isn't it our daily job ?